src/components/Dashboard/useDashboard.tsx

```
@@ -144,12 +148,36 @@
 
         // TODO: use history entry typing
-        requestHistory.entries.forEach((entry: any) => {
+        history.entries.forEach((entry: any) => {
+            let type: interfaces.IDashboardItem['type']
+            let amount: string
+            let tokenSymbol: string
+            let link: string
+            switch (entry.type) {
+                case 'SEND_LINK':
+                    //TODO: this will be rewritten with history service
+                    const password = getFromLocalStorage(`sendLink::password::${entry.uuid}`)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)